hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This patch adds a :ref:`test statistics <statistics>` event when a generated example is rejected via :func:`assume <hypothesis.assume>` or :func:`reject <hypothesis.reject>`.
```

Comment:
Docs build is failing because we don't document `reject()`; it's not worth adding a deprecation warning but I'd prefer to steer people to `assume(False)` - fewer functions makes for a simpler and more-discoverable API.

Reply:
I didn't realize we don't document reject at all! I agree that we should be pushing `assume(False)` in general instead.